src/fhir/CarePlan.DCW.CogLevel1.NoDementiaDx.json

```diff
@@ -0,0 +1,75 @@
+{
+    "resourceType": "CarePlan",
+    "id": "TBD",
+    "description": "DCW cognition domain, level 1 (which is only for patients who don't have a dementia Dx)",
+    "status": "active",
+    "intent": "order",
+    "category": [
+        {
+            "coding": [
+                {
+                    "system": "http://snomed.info/sct",
+                    "code": "719091000000102",
+                    "display": "Questionnaire"
+                }
+            ],
+            "text": "Questionnaire"
+        }
+    ],
```

Comment:
You'll see this identical block in every Questionnaire-related CarePlan, and the `code` within will be consistent.